.github/workflows/pr.yml

```diff
@@ -21,6 +21,23 @@ jobs:
       working-directory: charts/rabbitmq
       run: ./test.sh
 
+  test-all-examples:
+    name: test-all-examples
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+    - name: System tests
+      run: |
+        curl -s "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"  | bash
+        export GOPATH=$HOME/go
+        export PATH=$PATH:$GOPATH/bin
+        make install-tools
+        kind create cluster --image kindest/node:v1.19.1
```

Comment:
I could see an argument for doing this on all supported k8s versions, like the system tests do, in parallel. This might allow us to spot where a PR makes a specific use case of the operator no longer functional (though I could also see the argument that this would hopefully be caught in the system tests).

Reply:
The examples are tested as 'dry-runs' without testing specific functionality and only about api compatibility to the rabbitmqcluster CRD. The rabbitmqcluster CRD that's installed, whether to k8s `1.17` or `1.19`, is the same, so I don't see how testing against multiple k8s versions could increase test coverage.